### PR TITLE
Feature/marker

### PR DESCRIPTION
**Pull request by xIrusux**
just updated the marker color to red (like the gen change logo) and made the font in the pop up bigger.

### CODE REVIEW

**Comment by samjam48** [comment]
This could be useful to make a map marker of exact requirements https://leafletjs.com/examples/custom-icons/

**Comment by xIrusux** [comment]
hey sam 

I dont know how to change the marker to pastel green but feel free to do so yourself in a branch. 

can change the icon const name

File: src/components/ResultsPage/LeafletMap.js [hunk 1]
```diff
@@ -2,6 +2,18 @@ import React, { useEffect, useRef } from "react";
 import L from "leaflet";
 
 const LeafletMap = ({ searchLongLat, markersData }) => {
+  // marker style
+  var greenIcon = new L.Icon({
```
Comment (gminova) [inline src/components/ResultsPage/LeafletMap.js]: perhaps change to ES6

**Comment by samjam48** [comment]
okay tried it and tbh didn't like it with green haha.
now that airtable is sorted and confirmed that wasn't a bug related to this at all Ive merged with latest updates and then pulling into master

**Comment by samjam48** [comment]
> hey sam
> 
> I dont know how to change the marker to pastel green but feel free to do so yourself in a branch.
> 
> can change the icon const name

fyi - I just downloaded a map icon svg and changed the color

**Pull request merged**